runtime/doc/news.txt

```diff
@@ -52,6 +52,10 @@ NEW FEATURES                                                    *news-features*
 
 The following new APIs or features were added.
 
+• Neovim now supports a new environment variable named NVIM_APPNAME which
```

Comment:
nit: can leave out "Neovim now supports" in news.txt, else the items will be very repetitive